lib/Core/SQLiteBuildDB.cpp

```diff
@@ -327,7 +327,8 @@ class SQLiteBuildDB : public BuildDB {
     sqlite3_finalize(getKeysWithResultStmt);
     getKeysWithResultStmt = nullptr;
 
-    assert(sqlite3_close(db) == SQLITE_OK && "The database connection could not be closed. That means there are prepared statements that are not finalized, data blobs that are not closed or backups not finished.");
+    int result = sqlite3_close(db);
```

Comment:
You might need a ‘(void)result’ line after this to prevent warnings in no assert builds.

Reply:
Yep, you're right. Thanks